articles/quickstart/spa/vuejs/02-calling-an-api.md

```diff
@@ -0,0 +1,234 @@
+    "dev": "npm-run-all --parallel serve api",
+    "api": "node server.js"
+  },
+
+  // .. package dependencies and other JSON nodes
+}
+```
+
+You can now start the project using `npm run dev` in the terminal, and the frontend Vue.js application will start up alongside the backend API.
+
+### Set up a proxy to the backend API
+
+To call the API from the frontend application, the development server must be configured to proxy requests through to the backend API. To do this, add a `vue.config.js` file to the root of the project and populate it with the following code:
+
+```js
+// vue.config.js
+
+module.exports = {
+  devServer: {
+    proxy: {
+      "/api": {
+        target: "http://localhost:3001"
+      }
+    }
+  }
+};
+```
+
+::: note
+This assumes that your project was created using [Vue CLI 3](https://cli.vuejs.org/guide/). If your project was not created in the same way, the above should be included as part of your Webpack configuration.
+:::
+
+With this in place, the frontend application can make a request to `/api/external` and it will be correctly proxied through to the backend API at `http://localhost:3001/api/external`.
+
+### Restart the application
+
+If you are already running the application using `npm run serve`, then stop that process for now. Restart the app using the following command, which will run both the frontend app and the backend API together:
+
+```bash
+PORT=3000 npm run dev
+```
+
+## Add the Audience
+
+The API backend that you created above expects API calls to have a JSON Web Token attached so that it can authorize the call. To do this, you must instruct Auth0 to emit an access token in JWT format. This is done by supplying the API identifier when your users authenticate, telling Auth0 which API you want an access token for.
+
+Open `auth_config.json` in the root of the project and make sure that a value for `audience` is exported along with the other settings. The value for `audience` should be the "API identifier" for the API you created above:
+
+```json
+{
+  "domain": "${account.namespace}",
+  "clientId": "${account.clientId}",
+  "audience": "${apiIdentifier}"
+}
+```
+
+Finally, open `main.js` and configure the Auth0 plugin with this audience value:
+
+```js
+
+// .. other imports
+
+// NEW - import the audience
+import { domain, clientId, audience } from "../auth_config.json";
+
+// ..
+
+Vue.use(Auth0Plugin, {
+  domain,
+  clientId,
+  audience,   // NEW - configure the plugin with the audience value
+  onRedirectCallback: appState => {
+    router.push(
+      appState && appState.targetUrl
+        ? appState.targetUrl
+        : window.location.pathname
+    );
+  }
+});
+
+// .. more Vue configuration ..
+
+```
+
+## Call the API Using an Access Token
+
+The frontend Vue.js application should be modified to include a page that calls the API using an access token. Similar to the previous tutorial, this includes modifying the Vue router and adding a new view with a button that calls the API.
```

Comment:
Modify the frontend Vue.js application to include a page that calls the API using an access token. Like the previous tutorial, you'll need to change the Vue router and add a new view with a button that calls the API.